internal/test/playwright_test.go

```diff
@@ -121,14 +124,15 @@ func run(t *testing.T, command string) string {
 func daemonize(t *testing.T, command string) {
 	t.Helper()
 
-	shPath, err := exec.LookPath("sh")
-	if err != nil {
-		t.Fatalf("[unexpected] %v", err)
+	var cmd *exec.Cmd
+	if runtime.GOOS == "windows" {
+		cmd = exec.Command("cmd", "/C", "start", "/B", command)
+	} else {
+		cmd = exec.Command("sh", "-c", command+" &")
```

Comment:
This won't do what you think it will I don't think?

Reply:
Hmm, I tested it on windows/linux, and it worked fine. I think my only difference was the background (via & and /B), I can run the previous tests as well to double check they behave the same